src/time_integration/methods_SSP.jl

```diff
@@ -237,5 +237,7 @@ function Base.resize!(semi, volume_integral::VolumeIntegralSubcellLimiting, new_
     # Resize container subcell_limiter_coefficients
     @unpack limiter = volume_integral
     resize!(limiter.cache.subcell_limiter_coefficients, new_size)
+    (; alpha) = limiter.cache.subcell_limiter_coefficients
+    alpha .= zero(eltype(alpha))
```

Comment:
Why `resize!` somewhere else and then unpack it again here just to set it to zero?
Would be more readable if resizing and setting to zero would be in the same place.